### ISSUE TITLE
Calendar Month is announced as editable by Narrator

### ISSUE THREAD

**Issue by AkshanshSrivastava03** [OfficeDev/office-ui-fabric-react#5718]
### Bug Report

- __Package version(s)__: 5.112.1004

#### Priorities and help requested (not applicable if asking question):

Products/sites affected: (if applicable) VSTS

#### Describe the issue:
Open the Calendar and hover on the calendar months using Inspect. Using Inspect, Observe "Control type, IsTextEditPatternAvailable and IsTextPatternAvailble" properties for these buttons.

![image](https://user-images.githubusercontent.com/24951429/43391326-09aaa718-940e-11e8-8159-8651f203e845.png)

Due to this, the narrator announces each of the months as editable.
![image](https://user-images.githubusercontent.com/24951429/43391372-247dbb2a-940e-11e8-8cae-e3535c030aae.png)

#### Expected behavior:
IsTextEditPatternAvailable and IsTextPatternAvailble should be false

**Comment by dzearing** [comment]
I don't understand what is causing this or the work required to fix this and don't see comments here explaining anything. @lorejoh12 is there something easy to fix here? Is there an aria tag missing somewhere? @AkshanshSrivastava03 do you see an element with the wrong role or tag?

I am unclear on what action is required.

**Comment by lorejoh12** [comment]
it's the role=gridcell, I think by default gridcells are considered editable (even though clearly this isn't).

Should be a simple matter of adding aria-readonly="true" to the cells

**Comment by lorejoh12** [comment]
This was fixed in #8475
